packages/database/drizzle/schema.ts

```diff
@@ -493,6 +493,43 @@ export const contentLegalsLocalized = content.table(
   }),
 );
 
+// NEWSLETTER
+
+export const contentNewsletters = content.table('newsletters', (t) => ({
+  resourceId: t
+    .integer()
+    .primaryKey()
+    .notNull()
+    .references(() => contentResources.id, { onDelete: 'cascade' }),
+
+  id: t.uuid().unique().notNull(),
+  level: t.varchar({ length: 255 }),
+  author: t.text().notNull(),
+  websiteUrl: t.text(),
+  publication_date: t.text(),
+  title: t.text().notNull(),
+  tags: t.text().array(),
+  contributors: t.text().array(),
+  language: t.varchar({ length: 10 }).notNull(),
+}));
+
+export const contentNewslettersLocalized = content.table(
```

Comment:
For consistency with builders and buildersLocalized tables:

1. Add originalLanguage to contentNewsletters table for the original language.
2. Add language to contentNewslettersLocalized and include it in its primary key to support translations.

This ensures clear and consistent localization handling -> feel free to ask me if you need more informations

You will also have to make some changes in the sync and queries/services related to that